src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -322,9 +321,20 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionContext context)
                 spawnMetrics.setUploadTime(
                     uploadTime.elapsed().minus(networkTime.getDuration().minus(networkTimeStart)));
               }
+              final AtomicBoolean reportedExecuting =
```

Comment:
Put this on one line, please. Technically, the final modifier is unnecessary as of Java support for closures (Java 9?) which automatically capture effectively-final fields.